components/FundingPlatform/ApplicationView/ApplicationTab/ApplicationDataView.tsx

```diff
@@ -96,6 +96,16 @@ export const ApplicationDataView: FC<ApplicationDataViewProps> = ({ application,
       );
     }
 
+    // Handle Karma profile link fields (use field type from schema)
+    const fieldType = fieldKey ? fieldTypeMap[fieldKey] : undefined;
+    if (
+      fieldType === "karma_profile_link" &&
+      typeof value === "string" &&
+      /^0x[a-fA-F0-9]{64}$/.test(value)
```

Comment:
i see you using this regex in multiple places, maybe worth to have it stored in a helper and import it